Tests/TensorFlowTests/LayerTests.swift

```diff
@@ -1134,16 +1151,17 @@ final class LayerTests: XCTestCase {
         let grad = gradient(at: x, lnLayer) { $1($0).squared().sum() }
         // The expected values and gradients were computed using the following Python code:
```

Comment:
#499 
Change to "The expected value of the gradient was computed using the following Python code:"

Reply:
@rxwei this is not a part of this PR, this was already added when I merged into master.